### ISSUE TITLE
New version (0.7.0) failing with EV Subarus

### ISSUE THREAD

**Issue by villasenor** [G-Two/homeassistant-subaru#68]
I'm using the latest HACS version (v0.7.0). Seems like the timestamp that is returned for the EV Time to Full Charge isn't being parsed properly, which causes the entire integration to fail.

```

This error originated from a custom integration.

Logger: custom_components.subaru
Source: custom_components/subaru/remote_service.py:120 
Integration: Subaru (HACS) (documentation, issues) 
First occurred: December 5, 2022 at 10:40:20 PM (82 occurrences) 
Last logged: 11:13:54 AM

Unexpected error fetching subaru_data data: 'eventDateStr'
Traceback (most recent call last):
  File "/usr/src/homeassistant/homeassistant/helpers/update_coordinator.py", line 205, in _async_refresh
    self.data = await self._async_update_data()
  File "/usr/src/homeassistant/homeassistant/helpers/update_coordinator.py", line 164, in _async_update_data
    return await self.update_method()
  File "/config/custom_components/subaru/__init__.py", line 105, in async_update_data
    return await _refresh_subaru_data(hass, entry, vehicles, controller)
  File "/config/custom_components/subaru/__init__.py", line 199, in _refresh_subaru_data
    await refresh_subaru(vehicle, controller)
  File "/config/custom_components/subaru/remote_service.py", line 120, in refresh_subaru
    success = await controller.fetch(vin, force=True)
  File "/usr/local/lib/python3.10/site-packages/subarulink/controller.py", line 504, in fetch
    result = await self._fetch_status(vin)
  File "/usr/local/lib/python3.10/site-packages/subarulink/controller.py", line 977, in _fetch_status
    status = self._parse_condition(js_resp, vin)
  File "/usr/local/lib/python3.10/site-packages/subarulink/controller.py", line 1206, in _parse_condition
    keep_data[sc.EV_TIME_TO_FULLY_CHARGED_UTC] = data[api.API_TIMESTAMP] + timedelta(
KeyError: 'eventDateStr'

```

**Comment by G-Two** [comment]
This issue should now be fixed in [0.7.1](https://github.com/G-Two/homeassistant-subaru/releases/tag/v0.7.1). Please confirm if it works for you. Thanks for reporting.

**Comment by villasenor** [comment]
It works! You're really doing a huge service to the community by maintaining the Subaru code. Thank you so much!
